sktime/forecasting/var.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+
+"""Vector Auto Regressor."""
+__all__ = ["VectorAutoRegression"]
+__author__ = ["Taiwo Owoseni"]
+
+from statsmodels.tsa.api import VAR as _VAR
+from sktime.forecasting.base.adapters import _StatsModelsAdapter
+from sktime.forecasting.base._base import DEFAULT_ALPHA
+import numpy as np
+import pandas as pd
+
+
+class VectorAutoRegression(_StatsModelsAdapter):
```

Comment:
Do you prefer `VAR` or `VectorAutoRegression`?

Reply:
VAR is fine, but we could rename it VectoerAutoRegressor?

Reply:
I don't have a strong preference, but if I could choose, I would say `VAR`